core/src/main/java/uk/ac/bbsrc/tgac/miso/core/service/printing/Driver.java

```diff
@@ -127,6 +137,110 @@ public String encode(Barcodable barcodable) {
       }
       return sb.toString();
     }
+  },
+  ZEBRA_8363 {
+    // Square
+
+    @Override
+    public String encode(Barcodable b) {
+      StringBuilder sb = new StringBuilder();
+      sb.append("CT~~CD,~CC^~CT~\r\n");
+      sb.append("^XA\r\n");
+      sb.append("~TA000~JSN^LT0^MNW^MTT^PON^PMN^LH0,0^JMA^PR6,6^MD15^LRN^CI0\r\n");
+      sb.append("^MMT\r\n");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "^MMT\r\n" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)